core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -674,7 +674,7 @@ public interface MetricsProducer {
   String METRICS_SCAN_QUERY_SCAN_RESULTS = METRICS_SCAN_PREFIX + "query.results";
   String METRICS_SCAN_QUERY_SCAN_RESULTS_BYTES = METRICS_SCAN_PREFIX + "query.results.bytes";
   String METRICS_SCAN_SCANNED_ENTRIES = METRICS_SCAN_PREFIX + "query.scanned.entries";
-
+  String METRICS_SCAN_ZOMBIE_THREADS = METRICS_SCAN_PREFIX + "zombie.threads";
```

Comment:
We've been trying to keep the class javadoc in sync.

Reply:
Good catch, updated the javadoc in e78a249.